extension/scripts/features/no-confirm/ttItemMarketNoConfirm.js

```diff
@@ -56,10 +56,12 @@
 			const icon = item.find(".buy .buy-link");
 			if (!icon) continue;
 
+			icon.className = "yes-buy t-blue h bold m-left10 tt-modified";

```

Comment:
- t-blue 
 - h 
 - bold 
 - m-left10
 
These classes are all 

1. useless, as it's an icon and those change literally nothing, except for `m-left10`, which you negate with the css .....
2. Torn classes, which should not be used in v6 (yes-buy is an exception since it's required for the functionality)